okteto/manifest.yaml

```diff
@@ -0,0 +1,17 @@
+---
```

Comment:
add cert-manager dependency

Reply:
Okteto installs cert-manager as part of its installation. Do I need to list it here?

Reply:
I would say if possible better would be to add via dependency or have a mechanism  where if someone already has cert-manager installed then it gets skipped

Reply:
The downside of that is that the cert-manager version of the marketplace is a few versions behind  (1.0.3 vs 1.3.0), so the manifests might not work. Are there any plans to update that?

Reply:
it is 
kubectl apply -f https://github.com/jetstack/cert-manager/releases/download/v1.2.0/cert-manager.yaml
this version